bip-0141.mediawiki

```diff
@@ -120,9 +120,11 @@ The new rule is ''block cost'' ≤ 4,000,000.
 
 Sigops per block is currently limited to 20,000. We change this restriction as follows:
 
-''Sigop cost'' is defined. The cost of a sigop in traditional script is 4, while the cost of a sigop in witness program is 1.
+Sigops in the current pubkey script, signature script, and P2SH check script are counted at 4 times their previous value.
```

Comment:
Any reason for removing the sigop cost terminology? I thought it was useful to distinguish from the original sigops definition.